settings.gradle

```diff
@@ -1,5 +1,7 @@
 def modules = ['coral-hive', 'coral-pig', 'coral-presto','coral-spark','coral-schema']
 
+rootProject.name = "com.linkedin.coral"
```

Comment:
Don't think this is the right way to set group id. See 
https://github.com/linkedin/transport/blob/6fc9a9274c147e999b8b52ee286973a57bbbd7f2/build.gradle#L25

Reply:
foud the solution. group id.